src/edu/washington/escience/myria/operator/CSVFileScanFragment.java

```diff
@@ -181,142 +187,184 @@ protected TupleBatch fetchNextReady() throws IOException, DbException {
       if (parser.isClosed()) {
         break;
       }
+
+      if (!onLastRow) {
```

Comment:
How does this handle an empty file (or one that contains only skipped lines)? Won't we get a `NoSuchElementException`?